### ISSUE TITLE
many to many strategy

### ISSUE THREAD

**Issue by camel-chased** [balderdashy/sails-redis#57]
right now when I want to have many-to-many relationship I have a one table called `user_groups__group_users` that contains id of every record which is `user_groups__group_users:id:1`
when i want to get all user groups i must get all the records out there -that is not efficient
better way is to store two tables like `user:1:groups:id` and `group:1:users:id` then when i want to search user groups i need only indexKey `user:1:groups:id` and get members/groups id that i want to get

**Comment by Ryanc1256** [comment]
That is a good idea, would require a bit of an effort to change things up though, but yes it would speed joins (.populate) up a bit.

**Comment by sailsbot** [comment]
Thanks for posting, @camel-chased.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
